pkg/assets/models.go

```diff
@@ -0,0 +1,8 @@
+package assets
+
+type Metadata []MetadataField
+
+type MetadataField struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type MetadataField should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4860757)